README.md

```diff
@@ -12,41 +12,41 @@ Documentation
 -------------
 
 Documentation for Puppet and related projects can be found online at the
-[Puppet Docs site](https://docs.puppetlabs.com).
+[Puppet Docs site](https://puppet.com/docs).
 
 HTTP API
 --------
-[HTTP API Index](https://docs.puppetlabs.com/puppet/latest/reference/http_api/http_api_index.html)
+[HTTP API Index](https://puppet.com/docs/puppet/4.10/http_api/http_api_index.html)
```

Comment:
If I merge this to 4.10.x and then merge up, then later branches will point to 4.10. If you want to point to specific puppet versions then we probably need 2 PRs to 4.10 and 5.3 so that the merge up results in a merge conflict (assuming 5.3 is merged first).

Reply:
I think two separate PRs provide the least bad result since that index would be significantly different between 4.10 and 5.3.

Could I cherry-pick this commit to 5.3.x, then change the URL in another commit?

Reply:
Sure that'd work.